backend/queue-service/src/main/kotlin/com/ticketqueue/queue/service/ScheduleValidator.kt

```
@@ -0,0 +1,111 @@
+        }
+
+        // 2. Event Service 호출 (캐시 미스)
+        val response = try {
+            eventServiceClient.checkSellable(scheduleId)
+        } catch (e: BusinessException) {
+            if (e.errorCode == ErrorCode.RESOURCE_NOT_FOUND || e.errorCode == ErrorCode.SCHEDULE_NOT_FOUND) {
+                throw QueueException(ErrorCode.SCHEDULE_NOT_FOUND)
+            }
+            logger.error(e) { "Event service call failed (fail-closed): scheduleId=$scheduleId" }
+            throw QueueException(ErrorCode.INTERNAL_SERVER_ERROR)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)